app/javascript/components/common/MedianWaitTime.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import {
+  durationFromSeconds,
+  durationTimeElementFromSeconds,
+} from '../../utils/time'
+
+type MedianWaitTimeProps = {
+  seconds?: number
+}
+
+export function MedianWaitTime({ seconds }: MedianWaitTimeProps) {
```

Comment:
I've created a new component to render the median wait time, as we are displaying it in two locations and might want to use it in more.